.gitmodules

```diff
@@ -3,4 +3,5 @@
 	url = https://github.com/smartdevicelink/jsoncpp.git
 [submodule "tools/rpc_spec"]
 	path = tools/rpc_spec
-	url = https://github.com/smartdevicelink/rpc_spec.git
+	url = https://github.com/LuxoftSDL/rpc_spec.git
```

Comment:
Temporary commit? Should not be part of actual donation..

Reply:
@atiwari9 Yes, you are right, this is a temporary commit with the link to the rpc_spec that contains MOBILE API changes. We need it just for testing and it will be deleted before the merge

Reply:
@jacobkeeler Done in d420676